app.yaml

```diff
@@ -1,4 +1,5 @@
-application: "rdmprotocol-hrd"
+application: "olp-rdm-staging"
```

Comment:
we should really just implement [grunt-gae](/maciejzasada/grunt-gae), so we can define multiple targets and make a task that starts the server and the watch task with [concurrent](/sindresorhus/grunt-concurrent).

[also appcfg.py has -A for specifying the app-id](https://cloud.google.com/appengine/docs/python/tools/uploadinganapp#Python_Uploading_the_app)